http/internal/logic/social/followLogic.go

```diff
@@ -24,7 +32,78 @@ func NewFollowLogic(ctx context.Context, svcCtx *svc.ServiceContext) *FollowLogi
 }
 
 func (l *FollowLogic) Follow(req *types.FollowRequest) (resp *types.FollowResponse, err error) {
-	// todo: add your logic here and delete this line
-
-	return
+	UserClaims, err := utils.ParseToken(req.Token, l.svcCtx.Config.JwtAuth.Secret)
+	if err != nil {
```

Comment:
no need to handle this error, middleware already do it for you.

Reply:
userClaims, _ := utils.ParseToken(req.Token, l.svcCtx.Config.JwtAuth.Secret)